CloneResponsive/index.html

```diff
@@ -0,0 +1,432 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" type="text/css" href="./styles.css" />
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Inter:wght@200;300;400;500;600;700;800&display=swap"
+    rel="stylesheet">
+  <title>Clone</title>
+</head>
+
+<body>
+  <section class="main__container">
+    <section class="header">
+      <div class="logo">
+        <img src="./images/Logo.svg" alt="logo" class="logo__icon" />
+      </div>
+      </div>
+      <div class="header__main">
+        <div class="header__main__searchbar">
+          <div class="header__main__searchbar__icon">
+            <img src="./images/header-Icons/search.svg" alt="search icon">
+          </div>
+          <div class="header__main__searchbar__placeholder">
+            <input type="search" placeholder="Search..." id="search-input" size=100>
+          </div>
+        </div>
+        <div class="header__main__notification">
+          <div class="header__main__notification__icon">
+            <img src="./images/header-Icons/notification.svg" alt="notification indicator">
+          </div>
+        </div>
+        <div class="header__main__calendar">
+          <div class="header__main__calendar__icon">
+            <img src="./images/header-Icons/calendar.svg" alt="calendar icon">
+          </div>
+        </div>
+        <div class="header__main__tiles">
+          <div class="header__main__tiles__icon">
+            <img src="./images/header-Icons/tiles.svg" alt="app tiles">
+          </div>
+        </div>
+        <div class="header__main__avatar">
+          <div class="header__main__avatar__icon">
+
+            <img src="./images/header-Icons/green-indicator.svg" alt="status indicator"
+              class="header__main__avatar__icon__status">
+          </div>
+        </div>
+
+      </div>
+    </section>
+
+    <section class="content__container">
```

Comment:
Strange usage of BEM. Seems you don't understand it right. 
It is always `block_element` or `block_element__modifier`.
If you need several words in the block use dash `my-block_first-element__green-modifier`